pkg/apis/policy/v1alpha1/types_certificaterequestpolicy.go

```diff
@@ -37,19 +37,23 @@ var CertificateRequestPolicyKind = "CertificateRequestPolicy"
 // makes decisions on whether applicable CertificateRequests should be approved
 // or denied.
 type CertificateRequestPolicy struct {
-	metav1.TypeMeta   `json:",inline"`
-	metav1.ObjectMeta `json:"metadata,omitempty"`
+	metav1.TypeMeta `json:",inline"`
+	// +optional
+	metav1.ObjectMeta `json:"metadata"`
```

Comment:
In this KEP: https://github.com/kubernetes/enhancements/tree/master/keps/sig-api-machinery/2335-vanilla-crd-openapi-subset-structural-schemas#metadata, it looks like the metadata field is not required.

I think it is fine to always have this field in the marshalled resource.